Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -197,13 +211,27 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
             </TooltipHost>
           </div>
         </Flex.Item>
-        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} >
+        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '16%' }} >
+          <Text
+            truncated
+            className="semiBold"
+            content={message.sendingStartedDateTime}
+          />
+        </Flex.Item>
+        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '16%' }} >
           <Text
             truncated
             className="semiBold"
             content={message.sentDate}
```

Comment:
Make sure, if this is null, that it just shows nothing.

Reply:
if it is null, it will show nothing